src/components/schedule/channelList.js

```diff
@@ -1,17 +1,28 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import ChannelListItem from './channelListItem';
+import { FETCH_STATES } from '../constants';
 
-const ChannelList = ({ channels }) => (
+const renderList = channels => channels.map(channel =>
```

Comment:
Zmieściłoby się to w jednej linijce? Jak dla mnie byłoby nadal czytelne i wyglądałoby "krócej"